src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java

```diff
@@ -10,7 +10,7 @@
 
 public class AdsEvolveObject extends AdsBaseObject {
 
-  private static final String EVOLVE_SELECTOR = " script[src*=\"4403ad\"]";
+  private static final String EVOLVE_SELECTOR = " [id*='wikia_gpt/4403/ev/wikia_intl']";
```

Comment:
I thought about it but having it too strict isn't good either. Now the selector will work if somebody decides to change `<div>` to `<section>` ;)